ebean-datasource/src/main/java/io/ebean/datasource/pool/PooledConnection.java

```diff
@@ -424,11 +422,13 @@ public void close() throws SQLException {
 
       if (resetSchema) {
```

Comment:
I asked myself, if we can just use `if (currentSchema != null)` here...
But according to javadoc of `java.sql.Connection`, getSchema() can return null, so `connection.setSchema(null)` is a valid statement..., So we need booleans here